src/connector/src/sink/big_query.rs

```diff
@@ -61,6 +58,13 @@ pub struct BigQueryCommon {
     pub dataset: String,
     #[serde(rename = "bigquery.table")]
     pub table: String,
+    #[serde(rename = "bigquery.max_batch_rows", default = "default_max_batch_rows")]
+    #[serde_as(as = "DisplayFromStr")]
```

Comment:
do we really need to define `serde_as` here?

Reply:
Without this, the conversion from string to usize cannot be accomplished

Reply:
I see. I missed that part.